samples/snippets/activate_subordinate_ca.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START privateca_activate_subordinateca]
+import time
+
+import google.cloud.security.privateca_v1 as privateca_v1
+
+
+def activate_subordinate_ca(
+    project_id: str,
+    location: str,
+    ca_pool_name: str,
+    subordinate_ca_name: str,
+    pem_ca_certificate: str,
+    ca_name: str,
+) -> None:
+    """
+    Activate a subordinate Certificate Authority.
+    *Prerequisite*: Get the CSR of the subordinate CA signed by another CA. Pass in the signed
+    certificate and (issuer CA's name or the issuer CA's Certificate chain).
+    *Post*: After activating the subordinate CA, it should be enabled before issuing certificates.
+    Args:
+        project_id: project ID or project number of the Cloud project you want to use.
+        location: location you want to use. For a list of locations, see: https://cloud.google.com/certificate-authority-service/docs/locations.
+        ca_pool_name: set it to the CA Pool under which the CA should be created.
+        pem_ca_certificate: the signed certificate, obtained by signing the CSR.
+        subordinate_ca_name: the CA to be activated.
+        ca_name: The name of the certificate authority which signed the CSR.
+            If an external CA (CA not present in Google Cloud) was used for signing,
+            then use the CA's issuerCertificateChain.
+    """
+
+    caServiceClient = privateca_v1.CertificateAuthorityServiceClient()
```

Comment:
```suggestion
    ca_service_client = privateca_v1.CertificateAuthorityServiceClient()
```

Use snake case for python variables.  [PEP 8 Style Guide](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)